mark.c

```diff
@@ -619,6 +620,11 @@ GC_INNER mse * GC_mark_from(mse *mark_stack_top, mse *mark_stack,
   DECLARE_HDR_CACHE;
 
 # define SPLIT_RANGE_WORDS 128  /* Must be power of 2.          */
+/*!
```

Comment:
I'm not sure I understand this comment. Are we overriding `PTRS_TO_BYTES` or ... ?

Reply:
PTRS_TO_BYTES is a new macro. I did not want to override the macro WORDS_TO_BYTES everywhere, because I was not sure of the implications. I have moved this to a header file 
[include/private/gc_priv.h](https://github.com/capablevms/bdwgc/pull/1/files#diff-fb5b7993183e049f1b071ff0616d2a6fb36c7fbac5c2bab8d54d54227bef1841R858)

fixed in commit [408f667b](https://github.com/capablevms/bdwgc/pull/1/commits/408f667bb59d4d389e44ff70d5631bf922413ed9)

Reply:
> PTRS_TO_BYTES is a new macro. I did not want to override the macro WORDS_TO_BYTES everywhere, because I was not sure of the implications

I'm going to push back: since, so far, we seem to be always changing `WORDS_TO_BYTES` to `PTRS_TO_BYTES` it would surely be better to just override `WORDS_TO_BYTES` and see what happens, rather than creating a huge diff that will make future merges harder (as we're currently doing).